profiling/src/allocation.rs

```diff
@@ -122,12 +125,17 @@ pub fn allocation_profiling_rinit() {
         return;
     }
 
+    unsafe {
+        HEAP = Some(zend::zend_mm_get_heap());
+    }
```

Comment:
Should this get set to `None` in rshutdown after it's handlers have been reset?

Reply:
You are right